.gitignore

```diff
@@ -38,3 +40,8 @@ docs/api/*
 docs/_build/*
 cover/*
 MANIFEST
+pyvenv.cfg
+application.cfg
+
+#Mac auto-generated files
+**/.DS_Store
```

Comment:
This should not be part of **the repository**'s `.gitignore`:
Postgraas is in no way Mac-related, so you should capture something like this in [**your personal global .gitignore**](https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)
Otherwise we would have to also reflect all Linux and Windows related ignores ;-).

```suggestion
```

Reply:
You are right, I was not familiar with the global .gitignore. I just created one and removed this lines from the repository's .gitignore